DakoData/DakoData
# DAKO-163 BE API for company website contact columns

## Why do we need this PR?
* To support contact-us page to send email

## How did you address the issue?
* Add ContactEmail schema and endpoint
* Add ContactEmail MJML template (see https://mjml.io/try-it-live/7BtCN_prmVG)
* Set up GMail for SMTP

## How to test?
1. Create a gmail account and set up 2-factor verification
2. Get app password https://support.google.com/mail/answer/185833?hl=en
3. Set SMTP_USER and SMTP_PASSWORD to what you created
4. Run backend server locally
<br />  

> #### PR readiness check list
> - [v] Did it pass the Flake8/Standard JS check?
> - [v] Did you test this PR?  
> - [v] Did you explain the changes clearly?
> - [v] Does the PR title reference a Jira issue for the correct project (at the beginning of the title)?
